backend/src/main/java/com/soen341/instagram/service/impl/CommentService.java

```diff
@@ -0,0 +1,161 @@
+package com.soen341.instagram.service.impl;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.stereotype.Service;
+
+import com.soen341.instagram.dao.impl.AccountRepository;
+import com.soen341.instagram.dao.impl.CommentRepository;
+import com.soen341.instagram.dao.impl.PictureRepository;
+import com.soen341.instagram.exception.comment.CommentLengthTooLongException;
+import com.soen341.instagram.exception.comment.CommentNotFoundException;
+import com.soen341.instagram.exception.like.MultipleLikeException;
+import com.soen341.instagram.exception.picture.InvalidIdException;
+import com.soen341.instagram.exception.picture.PictureNotFoundException;
+import com.soen341.instagram.model.Account;
+import com.soen341.instagram.model.Comment;
+import com.soen341.instagram.model.Picture;
+
+@Service("commentService")
+public class CommentService
+{
+	@Autowired
+	private CommentRepository commentRepository;
+	@Autowired
+	private AccountRepository accountRepository;
+	@Autowired
+	private PictureRepository pictureRepository;
+	@Autowired
+	@Qualifier("UserDetailsService")
+	private UserDetailsService userDetailsService;
+
+	private static int maxCommentLength = 250;
```

Comment:
`final` for constants

Reply:
Seconded.

Reply:
Also constants should be in all caps with snake casing.